doc/html/standalone_HTML.manifest

```diff
@@ -0,0 +1,88 @@
+index.html
```

Comment:
Did you mean to commit this one - these manifests are of no use to anyone that I can see?

Not sure why they're generated at all to be honest... it's a docbook thing :(

Reply:
I just committed all the Quickbook generated files.  I can't see any particular use for the manifest.

I've Perhaps this should be added to the .gitignore?

I would also favour not committing the entire generated html and adding the /html folder to the 'gitignore.  I can see that it is convenient for other developers (or even users) who don't have to rebuild the docs.  But it is a PITA to have all these files that keep changing (and must be cluttering up GIThub).  Would generating a PDF version for 'users' be more convenient?  (It has the side effect of being more easily searchable: unless I know the right section, I haven't found how to search the html folder easily.  Once 'published' Google is our friend.